pkg/cmdutil/json_flags.go

```diff
@@ -138,8 +138,13 @@ func (e *exportFormat) Write(ios *iostreams.IOStreams, data interface{}) error {
 
 	w := ios.Out
 	if e.filter != "" {
-		return jq.Evaluate(&buf, w, e.filter)
-	} else if e.template != "" {
+		var jqOutput = &bytes.Buffer{}
+		if err := jq.EvaluateFormatted(&buf, jqOutput, e.filter, ios.IsStdoutTTY()); err != nil {
+			return err
+		}
+		buf = *jqOutput
```

Comment:
Ditto: let's undo this change and have EvaluateFormatted write straight to the original buffer instead of replacing it

Reply:
Given that `&buf` is being read by `json.EvaluateFormatted()` first, is this at all possible?

I guess I'll have to research how Go `bytes.Buffer` deals with being treated as a reader and as a writer with no intervening seek back to the start. :-) I had an unexamined assumption that that would be a problem.

Reply:
Ditto here; EvaluateFormatted now hides the extra formatting buffer, so this is resolved.